acceptance/dockerImage/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:trusty
+FROM ubuntu:xenial
```

Comment:
Two things
1. Again, we need to build/push these changes to a new version of the image, and that image has to be pushed before it can be used (which will require a change to the runner scripts to invoke a different image version)..
2. These changes need to go in `Dockerfile.xenial` because a couple of people are still on trusty.